yellowbrick/regressor/residuals.py

```diff
@@ -240,8 +250,9 @@ def finalize(self, **kwargs):
         self.ax.legend(loc='best', frameon=True)
```

Comment:
If we give the user the ability to change the alpha, do we want to use `manual_legend` here?

Reply:
Ideally yes --  unfortunately though, I think putting a `manual_legend` in `finalize()` will overwrite the legend created for the best-fit and 45 degree lines, which are currently plotted and created in the `draw()` method (since best-fit at least depends on knowing `y` and `y_pred`). Another slightly odd consequence is that instead of seeing the label for R squared as a circle, it will be a bar:

(no manual legend)
![image](https://user-images.githubusercontent.com/8760385/44630910-bb432980-a932-11e8-9247-64e77a46fb1a.png)

(with manual legend)
![image](https://user-images.githubusercontent.com/8760385/44630918-ce55f980-a932-11e8-8122-ed0f1c8b4e85.png)
   
I'm inclined to address this as a separate issue, but let me know if you have strong feelings.

Reply:
Ok, will do!

Reply:
Good comment!